pkgs/development/compilers/miranda/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl, gcc }:
+
+stdenv.mkDerivation rec {
+  version = "2.066";
+  pname = "miranda";
+
+  src = fetchurl {
+    url = "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-${builtins.replaceStrings [ "." ] [ "" ] version}-src.tgz";
+    sha256 = "1sqn031agwwgq3pka55pxd6m22dagmkxjhbwrni8gpigihg2h72j";
+  };
+
+  enableParallelBuilding = true;
+
+  configurePhase = ''
+    patchShebangs quotehostinfo
+  '';
+
+  patches = [ ./build.patch ];
```

Comment:
Instead of a single patch in the source tree, could you use `fetchpatch` to fetch the three files separately:

* `https://github.com/jtojnar/miranda/commit/be62d2150725a4c314aa7e3e1e75a165c90be65d.patch`
* `https://github.com/jtojnar/miranda/commit/048754606625975d5358e946549c41ae7b5d3428.patch`
* `https://github.com/jtojnar/miranda/commit/aea0a118a802a0da6029b781f7cfd388224263cf.patch`

and add a comment why each of them is necessary?